control-announcements/src/main/java/org/oskari/announcements/actions/AnnouncementsRestActionHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.oskari.announcements.actions;
+
+import fi.nls.oskari.control.ActionDeniedException;
+import fi.nls.oskari.control.ActionException;
+import fi.nls.oskari.control.RestActionHandler;
+import fi.nls.oskari.util.PropertyUtil;
+
+public class AnnouncementsRestActionHandler extends RestActionHandler {
+    private final static String PROPERTY_MODULES = "db.additional.modules";
+
+    public void requireAnnouncementsConfigured() throws ActionException {
```

Comment:
Why would this need to be checked? The database table is created by the core module anyways so it's always there. This can be removed.